amphibia-reptilia.csl

```diff
@@ -58,13 +63,13 @@
       </group>
     </layout>
   </citation>
-  <bibliography et-al-min="7" et-al-use-first="6">
+  <bibliography delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." name-as-sort-order="all" entry-spacing="2" line-spacing="2" hanging-indent="true">
     <sort>
       <key macro="author"/>
       <key macro="issued"/>
       <key variable="title"/>
     </sort>
-    <layout suffix=".">
+    <layout prefix="•&#9;" suffix=".">
```

Comment:
@RAWes, yes, it almost always helps to look at one or two recent published papers (the one I picked is from the "Free Sample Issue" linked to at http://www.brill.com/amphibia-reptilia).

(also, feel free to write in Dutch if you prefer :) )